build-profiling-ffi.sh

```diff
@@ -140,7 +140,6 @@ FEATURES=(
     "cbindgen"
     "crashtracker-collector"
     "crashtracker-receiver"
-    "data-pipeline-ffi"
```

Comment:
We would like to keep this as default feature shipped. It seems like headers issue is unrelated to what is included in the features.

Reply:
Re-added